src/components/About/AboutDetails.module.css

```diff
@@ -0,0 +1,74 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+.container {
+  background: var(--midnight-green);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 100%;
+  color: var(--primary-white);
+  padding-inline: 24px;
+  text-align: center;
+  gap: 16px;
+  position: relative;
+  padding-top: 80px;
+  padding-bottom: 108px;
+}
+
+.container::after {
+  content: "";
+  position: absolute;
+}
+
+.container::after {
+  width: 200px;
+  height: 100px;
+  bottom: 0px;
+  right: 0px;
+  transform: translate(50%, 0);
+  background-image: url("../../assets/bg-pattern-about-1-mobile-nav-1.svg");
+}
+
+@media (min-width: 768px) {
+  .container {
+    gap: 24px;
+    padding-top: 122px;
+    padding-bottom: 112px;
+  }
+  .aboutus {
+    width: 457px;
```

Comment:
Don't fix the width. This causes issues like the overflow at some screen sizes and various other potential issues with responsiveness.

You can set a max-width, or set width as a percentage of viewport width ([viewport units](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units)) or you can use grid or flex to set limits on the layout without fixing it exactly. There may be other options. What is your goal when you set the width here?